usb: netusb: Align NETUSB_MTU & wMaxSegmentSize and increase it to 1522

ubicore opened the pull request:
I saw a misalignment problem between the declared ECM wMaxSegmentSize and the NETUSB_MTU.
As rx_buf & tx_buf are defined with a NETUSB_MTU size, this cause that the frames over 1500 bytes can not be received and are discarded.

Secondly, as fragmentation is not supported (for UDP and TCP datagram), I suggest to increase the  NETUSB_MTU to the maximum of the possible ethernet frame size :1522

finikorg reviewed (approved):
LGTM with provision you fix issues with Signed-off-by, it is also better to rename Subject to:
`usb: netusb: Align NETUSB_MTU & wMaxSegmentSize and increase it to 1522`

ubicore commented:
I have added my gpg key used to sign the commit to my account.
But I don't knows how to relaunch the check

jukkar commented:
> But I don't knows how to relaunch the check

Just force push your branch to github.

finikorg commented:
Probably this is the issue:
```
65054ecfe6afac575a3aef8cf9a3d620a6f8e9b1: author email (nlantz nicolas.lantz@ubicore.net) needs to match one of the signed-off-by entries.
65054ecfe6afac575a3aef8cf9a3d620a6f8e9b1: author email (nlantz nicolas.lantz@ubicore.net) does not follow the syntax: First Last .
```

ubicore committed:
usb: netusb: Align NETUSB_MTU & wMaxSegmentSize and increase it to 1522

As rx_buf & tx_buf are defined with a NETUSB_MTU size, this cause
that the frames over 1500 bytes can not be received and are discarded.

Secondly, as fragmentation is not supported (for UDP and TCP datagram),
increase NETUSB_MTU to the maximum possible ethernet frame size :1522

Signed-off-by: Nicolas LANTZ <nicolas.lantz@ubicore.net>

ubicore commented:
Yes, thanks,

Looking here :
https://github.com/zephyrproject-rtos/ci-tools/blob/master/scripts/check_compliance.py#L311

I understood that I had changed my name of author between the commit and 
the edition for signature.

Solution, was to doing a :
git commit --amend --reset-author --no-edit

Le 30/01/2019 à 09:49, Andrei Emeltchenko a écrit :
>
> Probably this is the issue:
>
> |65054ecfe6afac575a3aef8cf9a3d620a6f8e9b1: author email (nlantz 
> nicolas.lantz@ubicore.net) needs to match one of the signed-off-by 
> entries. 65054ecfe6afac575a3aef8cf9a3d620a6f8e9b1: author email 
> (nlantz nicolas.lantz@ubicore.net) does not follow the syntax: First 
> Last . |
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/zephyrproject-rtos/zephyr/pull/12811#issuecomment-458861342>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AoBMEHGmAhhB_dsEd1lOYB6YN8HLIpILks5vIVx-gaJpZM4aXlgy>.
>

jukkar commented:
The commit message is still wrong, please fix it. Note that it is not enough to edit the github "title" field.

finikorg commented:
Do not make it toot complex, just make 
```
Author: nlantz <nicolas.lantz@ubicore.net>
```
and
```
    Signed-off-by: Nicolas LANTZ <nicolas.lantz@ubicore.net>
```
be the same (AFAIK)

jukkar reviewed (dismissed):
The change itself is good, just edit the commit message as suggested by @finikorg and force push your branch.
